projects/standard-rulesets/package.json

```diff
@@ -20,13 +20,11 @@
   },
   "dependencies": {
     "@stoplight/spectral-core": "^1.8.1",
-    "@stoplight/spectral-ruleset-bundler": "^1.5.0",
     "@stoplight/spectral-rulesets": "^1.14.1",
```

Comment:
Correct me if I'm wrong, but we can remove spectral-core and rulesets from standard rulesets? We just need to keep it around in the `rulesets-base` package for now